.travis.yml

```diff
@@ -8,12 +8,9 @@ env:
 before_install:
   - sudo apt-get update
   - sudo apt-get install gfortran liblapack-pic liblapack-dev
-  # Print NumPy version that is already installed by Travis CI:
-  - python -c "import numpy; print(numpy.__version__)"
```

Comment:
I think keeping this for debugging purposes is still a good idea (unless it does not work).

Reply:
ok cool. Nope it works. I moved it to the end of `before_install` just in case the other packages cause numpy to get updated.